### PR TITLE
Fix graphiql template

### PR DESCRIPTION
**Pull request by ThirVondukr**
## Types of Changes

- [ ] Core
- [x] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Documentation

### CODE REVIEW

**Comment by botberry** [comment]
Hi, thanks for contributing to Strawberry 🍓!

We noticed that this PR is missing a `RELEASE.md` file. We use that to automatically do releases here on GitHub and, most importantly, to PyPI!

So as soon as this PR is merged, a release will be made 🚀.

Here's an example of `RELEASE.md`:

```markdown
Release type: patch

Description of the changes, ideally with some examples, if adding a new feature.
```

Release type can be one of patch, minor or major. We use [semver](https://semver.org/), so make sure to pick the appropriate type. If in doubt feel free to ask :)


Here's the tweet text: 
```
🆕 Release (next) is out! Thanks to Doctor for the PR 👏

Get it here 👉 https://github.com/strawberry-graphql/strawberry/releases/tag/(next)
```

File: strawberry/utils/graphiql.py [hunk 1]
```diff
@@ -11,6 +11,8 @@ def get_graphiql_html(
     template = path.read_text(encoding="utf-8")
 
     if replace_variables:
-        template.replace("{{ SUBSCRIPTION_ENABLED }}", json.dumps(subscription_enabled))
+        template = template.replace(
+            "{{ SUBSCRIPTION_ENABLED }}", json.dumps(subscription_enabled)
+        )
```
Comment (patrick91) [inline strawberry/utils/graphiql.py:16]: this was a silly mistake! thanks for fixing it!

**Review by patrick91 — APPROVED** [review]
Thank you ♥️ I'll add a release file and merge 😊

**Comment by botberry** [comment]
Thanks for contributing to Strawberry! 🎉 You've been invited to join
the Strawberry GraphQL organisation 😊

You can also request a free sticker by filling this form: https://forms.gle/dmnfQUPoY5gZbVT67

And don't forget to join our discord server: https://strawberry.rocks/discord 🔥

**Pull request merged**